specification/azurefleet/AzureFleet.Management/examples/2024-05-01-preview/Fleets_CreateOrUpdate.json

```diff
@@ -383,7 +383,7 @@
     "200": {
       "body": {
         "properties": {
-          "provisioningState": "Creating",
```

Comment:
Same question, I thought we'd only change the latest?

Reply:
I corrected it, as it was wrong state  for 200 - 200 should be terminal state. @SahilArora92 suggested to correct it.